### PR TITLE
301 curriculum page

### LINKED ISSUE #301: Build redesigned Curriculum Page
We've discussed the manifesto + Sam's article (https://docs.google.com/document/d/1tzrPcd1bgBPHVK-Enj5BMpwlf08UT9uFe0m3we2ITII/edit#heading=h.5dfykppw6g2d) as well as the letter I wrote a year ago that students receive.

Can we compile this into something nice for the Curriculum page? If there's an appeal for this, I don't mind making a wireframe with pen/paper

### PR DESCRIPTION
**Pull request by sjmog**
Closes #301 

Redesigns the curriculum page to be more manifesto-y, more interactive, and just better. Includes CSS fixes and abstractions.

### CODE REVIEW

**Comment by sjmog** [comment]
@rubenkostucki is invited to contribute to the feedback here!

**Comment by sjmog** [comment]
*DO NOT MERGE* until the blog is up and the blog article links are fixed on this page

File: Gemfile.lock [hunk 1]
```diff
@@ -175,6 +175,3 @@ DEPENDENCIES
   rspec
   sass
   timecop
-
```
Comment (pitchinvasion) [inline Gemfile.lock]: ARGHGHGHGHGHHGHGHGHGHHGHGHGHGHGHGHGHGHGHGHGHGHGHGHGHJGHGHGHG

Comment (pitchinvasion) [inline Gemfile.lock]: Samm  could you do `gem update bundler` when in the site-mvp directory, you've got an old version of bundler in there

File: source/partials/curriculum/_manifesto.html.haml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+%section.article.article--centered.no-padding-bottom
+  %article
+    %h4.leader Our
+    %h3 Manifesto
+    %p
+      %span Learning is so much more than a curriculum and some educational materials.
```
Comment (pitchinvasion) [inline source/partials/curriculum/_manifesto.html.haml]: What's the span for?

File: source/partials/curriculum/_manifesto.html.haml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          %h3 Learning > Doing
+        %p You can read about riding a bike 'til the cows come home; unless you actually try riding one, you're unlikely to be competing in the Tour de France any time soon.
+      .column
+        .header-container
+          %h4.leader The Best Learning == 
+          %h3 Self-Guided Learning
+        %p It doesn't matter how much you pay a personal trainer - you still have to do the squats! As with training, so with learning.
+      .column
+        .header-container
+          %h4.leader Healthy Body &
```
Comment (pitchinvasion) [inline source/partials/curriculum/_manifesto.html.haml]: I might have missed the discussion here but two ampersands is more Ruby-codery no? Although I guess & is also a code thing

File: source/partials/curriculum/_manifesto.html.haml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          %h3 Healthy Mind
+        %p You have to be well to learn well.
+
+    .button-row
+      %p
+        = link_to "Read more about our Manifesto", "#blog-link", class: "button--horizontal"
+    
+
+   
+
```
Comment (pitchinvasion) [inline source/partials/curriculum/_manifesto.html.haml]: Bit o' whitespace here

File: source/sass/_definition_lists.scss [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        margin-bottom: 2em;
+      }
+      &:last-of-type p {
+        margin-bottom: 0;
+      }
+    }
+
+    @include media($desktop) {
+      display: table-cell;
+      width: 66.66667%;
```
Comment (pitchinvasion) [inline source/sass/_definition_lists.scss]: what's going on here? Isn't this a column with a width of 8, with another column somewhere else with a width of 4?

File: source/partials/curriculum/_manifesto.html.haml (hunk 2 above)
Comment (pitchinvasion) [inline source/partials/curriculum/_manifesto.html.haml]: Oh it's a leader I see now, maybe add a class so this is a little more clear?

**Comment by pitchinvasion** [comment]
Mobile alignment:

image

**Comment by pitchinvasion** [comment]
image

**Comment by pitchinvasion** [comment]
More mobile alignment issues

**Comment by pitchinvasion** [comment]
Generally looks good I particularly love the day in the life of a makers but looks like there's a few minor things to sort before squirrel

File: source/partials/curriculum/_manifesto.html.haml (hunk 2 above)
Comment (sjmog) [inline source/partials/curriculum/_manifesto.html.haml]: yeees will do as part of css refactor

File: source/partials/curriculum/_manifesto.html.haml (hunk 3 above)
Comment (sjmog) [inline source/partials/curriculum/_manifesto.html.haml]: I'll do `&&`

File: source/sass/_definition_lists.scss (hunk 5 above)
Comment (sjmog) [inline source/sass/_definition_lists.scss]: Yes, but it needed to be forced b/c the mixin was having specificity issues. Again, hopefully TBD in the css refactor

**Comment by sjmog** [comment]
Thank @pitchinvasion - the mobile alignment on the table is intended (less length in tables & weird line lengths)

The bastard final column alignment just won't do what I tell it to, no matter how I try. I'm gonna push it now and then I'll fix it as part of refactoring the grid system we're using :)

**Pull request merged**